greymatter-io/terraform-greymatter-ecs
# Ingress certs

closes #26 
closes #29 

This PR adds ingress certs on the edge load balancer and cleans up security groups.

The previous behavior was you could access edge lb at `http://{edge-dns}:10808/services/control-api/latest` and now it will use https and port 443 - `https://{edge-dns}/services/control-api/latest`.

The previous security group behavior was that a new sg was created for every sidecar - now one is created and will be reused (since they will all use the same port).

To test:

follow the instructions in docs/install.md to install, then verify that you can (only) connect to the edge control-api route with https at `https://{edge-dns}/services/control-api/latest`

If you already have a test file (bs?) just change the source to `ref=ingress_certs`.
